src/ModuleList/Quiz/Quiz.tsx

```diff
@@ -1,22 +1,78 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { useParams } from 'react-router-dom';
+import { SimpleGrid, Box, Flex } from '@chakra-ui/core';
+import { SubmitQuiz, CheckIcon, IncorrectIcon } from './styled';
+import { useQuizMeta } from 'src/common/api/curriculum/quiz';
+import { Question } from 'src/common/api/curriculum/quiz/requests';
 
 interface Params {
-  moduleId: string;
   quizId: string;
 }
 
+interface StringMap {
+  [key: string]: boolean;
+}
+
 const Quiz = () => {
-  const { moduleId, quizId } = useParams<Params>();
+  const { quizId } = useParams<Params>();
+
+  const { data } = useQuizMeta(Number(quizId));
+
+  const [questionIndexesAnsweredCorrectly, setQuestionIndexesAnsweredCorrectly] = useState<StringMap>({});
+  const [quizIsSubmitted, setQuizIsSubmitted] = useState<boolean>(false);
+
+  const handleAnsweredQuestion = (q: Question, qIndex: number, optionIndex: number) => {
+    return (e: React.ChangeEvent<any>) => {
+      questionIndexesAnsweredCorrectly[qIndex] = q.answer === optionIndex;
+      setQuestionIndexesAnsweredCorrectly(questionIndexesAnsweredCorrectly);
+    };
+  };
+
+  const checkQuiz = () => {
+    setQuizIsSubmitted(true);
+  };
+
+  const getLabelStyle = (q: Question, qIndex: number, optionIndex: number) => {
+    let color = '#FFFFFF';
+    if (quizIsSubmitted &&
+      !!questionIndexesAnsweredCorrectly[qIndex] === false &&
+      q.answer === optionIndex) {
+      color = '#C5A3B4';
+    }
+    return { color };
+  };
 
   return (
     <div>
-      <h1>
-        Module: {moduleId}
-      </h1>
-      <h2>
-        Quiz: {quizId}
-      </h2>
+      <SimpleGrid columns={1} spacing={10}>
+        {Array.isArray(data) && data.map((q: Question, qIndex: number) => (
+          <Box p={5} color="white" key={qIndex}>
+            <Flex>
+              {quizIsSubmitted && (questionIndexesAnsweredCorrectly[qIndex] ?
+                <CheckIcon color="green.500"/> : <IncorrectIcon color="red.500" />)}
+              <p>{ `${qIndex + 1}. ${q.question}`}</p>
+            </Flex>
+            <SimpleGrid columns={2} spacingX={1}>
+              {q.options.map((option, optionIndex) => (
+                <div key={optionIndex}>
+                  <input
+                    type="radio"
+                    id={option}
+                    name={qIndex.toString()}
+                    value={option}
+                    onChange={handleAnsweredQuestion(q, qIndex, optionIndex)}
+                    style={{margin: '5px'}}
+                  />
+                  <label htmlFor={option} style={getLabelStyle(q, qIndex, optionIndex)}>{option}</label>
+                </div>
+              ))}
+            </SimpleGrid>
+          </Box>
+        ))}
+      </SimpleGrid>
+      <Flex>
+        <SubmitQuiz onClick={checkQuiz}>Submit Quiz</SubmitQuiz>
+      </Flex>
```

Comment:
A couple of visual changes...

- Can we horizontally center the quiz?
![image](https://user-images.githubusercontent.com/22752497/100694350-7fd70700-3354-11eb-9f70-2d551dd2f7bd.png)

- Can we add more padding to the bottom after the submit button?
![image](https://user-images.githubusercontent.com/22752497/100694389-99784e80-3354-11eb-9545-2ce889da42f0.png)